src/api/db/models/Image.js

```diff
@@ -774,6 +790,52 @@ export class ImageModel {
     if (err instanceof GraphQLError) throw err;
     throw new InternalServerError(err);
   }
+
```

Comment:
Wasn't sure if there was a way to put a watch on certain functions, so far I have only added it to methods I thought were relevant

Reply:
I was just a bit unsure if there was a more automatic way to attach `updateReviewStatus` to our existing functions besides manually calling at the end of each operation. Not a huge deal